### DISCUSSION TITLE
Sugestão de curso

### DISCUSSION THREAD

**Discussion by Bruno-Piragine** [backend-br/forum#120] [Q&A] [answered]
Boa tarde
Sou iniciante e gostaria de uma indicação de curso que realmente seja bom, pois na Internet existe muita propaganda, gostaria de indicações de pessoas reais.
E qual área da programação vale mais a pena pra quem está iniciando, relacionado a oportunidades e salário. 
Desde já agradeço

**Comment by reginaldojunior** [comment, marked as answer]
Olá Bruno, blz?

Cara existe muita coisa na área de tecnologia, tem um mercado bem aquecido na área de desenvolvimento web e mobile. É uma área para se olhar com atenção. Mas a tecnologia não acaba nisso, somente dentro destas duas áreas a várias coisas que pode se feita, como desenvolvimento, QA, scrum master, Product Manager, UX entre várias outras possibilidades. Acredito como você está nesse forum você já tenha um interesse na área de programação.

Cursos você pode encontrar bacanas na Udemy lá tem de tudo. Um ótimo curso base que indico para todos que quer começar no mundo da programação são os do Guanabara, a didática dele é impecável. Você pode ver todos os cursos disponiveis neste link gratuitamente pelo youtube.

https://www.youtube.com/c/CursoemV%C3%ADdeo/playlists

Recomendo olhar o de lógica de programação, HTML e CSS, e também algum de uma linguagem backend, como python.

Um toque que dou é tentar ao máximo fugir destas promessas milagrosas de "vire dev em 7 dias", "como aumentar seu salário e ganhar 10 mil reais" o processo ele é longo, mas é com certeza certo. Se você fazer os cursos, se dedicar e estudar com certeza irá conseguir um espaço na área.

Por fim, indico esses podcasts do QuebraDev que da várias dicas para quem tá começando.

- https://quebra.dev/vire-programador-em-7-dias/
- https://quebra.dev/carreiras-em-ti-com-andreza-deza/
- https://quebra.dev/a-corrida-pelos-devs-realidade/
- https://quebra.dev/por-que-desenvolvimento-web/
- https://quebra.dev/iniciando-no-desenvolvimento-web-back-end/
- https://quebra.dev/iniciando-no-desenvolvimento-web-front-end/
- https://quebra.dev/periferia-e-a-comunidade/

**Comment by eltinhosp-eng** [comment]
Cara boa noite, 
também sou iniciante, comecei com estrutura de dados e algoritmos e hoje estou me aprofundando em Python...pois me identifiquei mais com a parte de backend...
Tem dois cursos que gosto bastante que do Guanabara e da Dio(Digital Innovation One os dois são gratuitos...creio que vc irá gostar.
Caso vc queira algo para trabalhar em nuvem, segue o link de um curso gratuito muito bom de um amigo, tem o curso de AWS - Cloud show de bola com certificado do basico ao avançado.
https://cabecananuvem.com/

segue os links:
. https://www.cursoemvideo.com/
. https://www.dio.me/
Mas uma dica, começa com estrutura de dados e algoritmos primeiro...depois vc entra na linguagem de programação, tanto backend ou frontend vai precisar.
